src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -109,28 +251,47 @@ void Interaction::execRef(dnnl::stream strm) {
     {DNNL_ARG_SRC, inputMemPtr->GetPrimitive()},
     {DNNL_ARG_WEIGHTS, inputMemPtr->GetPrimitive()},
     {DNNL_ARG_DST, outputMemPtr->GetPrimitive()}};
-
+    float* scales = fuseFQ ? fqScales.data() : nullptr;
     for (int64_t start = 0; start < batchSize; start++) {
```

Comment:
is there some reason to not use parallel_for?

Reply:
The reason here is that the DLRM model runs in single core, for simplification, I don't use `parallel_for`